mlflow/metrics/genai/metric_definitions.py

```diff
@@ -27,7 +27,9 @@ def correctness(
 
     An MlflowException will be raised if the specified version for this metric does not exist.
 
-    :param model: (Optional) The model that will be used to evaluate this metric. Defaults to GPT-4.
+    :param model: (Optional) The model that will be used to evaluate this metric. Defaults to
+        GPT-4. Your use of a third party LLM service (e.g., OpenAI) for evaluation may be subject
```

Comment:
Is the default here as gpt-4 or `openai:/gpt-3.5-turbo-16k`?

Reply:
its gpt-3.5. ill update it in this PR.